test/unittests/execute_floating_point_test.cpp

```diff
@@ -51,25 +51,25 @@ TYPED_TEST(execute_floating_point_types, test_values_selftest)
     EXPECT_EQ(std::size(TV::positive_all()), TV::num_positive + TV::num_nans);
 
     for (const auto nan : TV::positive_nans())
-        EXPECT_TRUE(std::isnan(nan));
+        EXPECT_TRUE(nan.is_nan());
```

Comment:
Shouldn't this change be part of the `is_nan` commit?

Reply:
This is changed because test values are `FP` objects now.